scripts/sct_propseg.py

```diff
@@ -315,15 +372,22 @@ def get_parser():
         else:
             sct.printv('\nERROR: the viewer has been closed before entering all manual points. Please try again.', verbose, type='error')
 
+    cmd += ' -centerline-binary'
     sct.run(cmd, verbose)
 
-    sct.printv('\nDone! To view results, type:', verbose)
     # extracting output filename
     path_fname, file_fname, ext_fname = sct.extract_fname(input_filename)
-    output_filename = file_fname+"_seg"+ext_fname
+    output_filename = file_fname + "_seg" + ext_fname
+
+    fname_centerline = file_fname + '_centerline' + ext_fname
+    check_and_correct(folder_output + output_filename, folder_output + fname_centerline)
+
+    # remove temporary files
+    sct.run('rm -rf ' + folder_output + 'tmp.*')
```

Comment:
Sorry, I did not notice that it is a list of tmp files.

You can do this instead.

from glob import glob

    for tmp_file in glob('tmp*'):
        os.remove(tmp_file)